coral/src/app/features/approvals/schemas/components/SchemaApprovalsTable.tsx

```diff
@@ -12,84 +13,92 @@ interface SchemaRequestTableData {
-    headerName: "Approve",
-    width: 30,
-    UNSAFE_render: (row) => {
-      return (
-        <GhostButton
-          onClick={() => alert("Approve")}
-          aria-label={`Approve schema request for ${row.topicname}`}
-          icon={tickCircle}
-        />
-      );
-    },
-  },
-  {
-    type: "custom",
-    // @TODO PR open in DS to be able to
-    // add an "invisible" header name that
-    // is used as aria-label. That will also
-    // solve the duplicate key warning
-    //https://github.com/aiven/design-system/pull/950
-    headerName: "Decline",
-    width: 30,
-    UNSAFE_render: (row) => {
-      return (
-        <GhostButton
-          onClick={() => alert("Decline")}
-          aria-label={`Decline schema request for ${row.topicname}`}
-          icon={deleteIcon}
-        />
-      );
-    },
-  },
-];
-
 type SchemaApprovalsTableProps = {
   requests: SchemaRequest[];
+  setDetailsModal: Dispatch<
+    SetStateAction<{
+      isOpen: boolean;
+      req_no: number | null;
+    }>
+  >;
 };
 function SchemaApprovalsTable(props: SchemaApprovalsTableProps) {
-  const { requests } = props;
-
+  const { requests, setDetailsModal } = props;
+  const columns: Array<DataTableColumn<SchemaRequestTableData>> = [
+    { type: "text", field: "topicname", headerName: "Topic" },
+    { type: "text", field: "environmentName", headerName: "Environment" },
+    { type: "text", field: "username", headerName: "Requested by" },
+    {
+      type: "text",
+      field: "requesttimestring",
+      headerName: "Date requested",
+      formatter: (value) => {
+        return `${value} UTC`;
+      },
+    },
+    {
+      type: "custom",
+      // @TODO PR open in DS to be able to
+      // add an "invisible" header name that
+      // is used as aria-label. That will also
+      // solve the duplicate key warning
+      //https://github.com/aiven/design-system/pull/950
+      headerName: "Details",
+      width: 30,
+      UNSAFE_render: (row) => {
+        return (
+          <GhostButton
+            onClick={() => setDetailsModal({ isOpen: true, req_no: row.id })}
+            icon={infoSign}
+            dense
+          >
+            <span aria-hidden={"true"}>View details</span>
+            <span className={"visually-hidden"}>
+              View schema request for {row.topicname}
+            </span>
+          </GhostButton>
+        );
+      },
+    },
+    {
+      type: "custom",
+      // @TODO PR open in DS to be able to
+      // add an "invisible" header name that
+      // is used as aria-label. That will also
+      // solve the duplicate key warning
+      //https://github.com/aiven/design-system/pull/950
+      headerName: "Approve",
+      width: 30,
+      UNSAFE_render: (row) => {
+        return (
+          <GhostButton
+            onClick={() => alert("Approve")}
+            aria-label={`Approve schema request for ${row.topicname}`}
+            icon={tickCircle}
```

Comment:
I think to maintain visual consistency, the icon should be a child of the Button, for approve and reject, like in the other tables. But that may be done at another point :o

Reply:
🤔  But that's something I recently changed to this way (not sure if here or in topics) to this approach. Or did we change that again because we are showing different icons dependent on the loading state we need for approvals?

Reply:
As I see it, to keep button sizes consistent:
- the icon should go in the `icon` prop for the Show details column (I think this is what you changed)
- the icon should be an `Icon` child component in the Approve and Reject columns

Otherwise, the approve and reject button look quite big, and also have a right margin that throws off their alignment

Reply:
Comparison:
- the way it is rendered in this PR (disregard the spacing, it's because of the header text that is not in other tables)
<img width="276" alt="Screenshot 2023-02-23 at 15 34 05" src="https://user-images.githubusercontent.com/20607294/220937442-41235959-7450-4de7-992f-b106beb41da8.png">
- the way I think it should be
<img width="212" alt="Screenshot 2023-02-23 at 15 34 16" src="https://user-images.githubusercontent.com/20607294/220937545-2baeae2d-13f0-4790-9025-79e0de9e4dcb.png">

Reply:
Removing thje header text, I think the misalignmebnt due to the margin that exists even when GhostButton has no text is visible
<img width="252" alt="Screenshot 2023-02-23 at 15 36 42" src="https://user-images.githubusercontent.com/20607294/220937950-26f60cc9-ce23-4c7c-b8ac-424f5531dcda.png">

Reply:
💡  OK, thanks for clarifying! Yes, we probably talked about adding the icon as prop for the _one_ column and I did go overboard and did it for all 😂  

I'll change that when adding the approve/reject stuff, then I need to touch that area anyways.